packages/cli/src/commands/follow.js

```diff
@@ -4,10 +4,12 @@ import os from 'os';
 import { E } from '@endo/far';
 import { makeRefIterator } from '@endo/daemon';
 import { withEndoAgent } from '../context.js';
+import { parsePetNamePath } from '../pet-name.js';
 
 export const followCommand = async ({ name, agentNames }) =>
   withEndoAgent(agentNames, { os, process }, async ({ agent }) => {
-    const iterable = await E(agent).lookup(name);
+    const namePath = name && parsePetNamePath(name);
+    const iterable = await E(agent).lookup(...namePath);
```

Comment:
`name` is required for the `follow` command:
```suggestion
    const iterable = await E(agent).lookup(...parsePetNamePath(name));
```